packages/frontend/app/components/curriculum-inventory/new-sequence-block.hbs

```diff
@@ -46,9 +46,9 @@
               {{t "general.level"}}:
               {{this.course.level}},
               {{t "general.startDate"}}:
-              {{moment-format this.course.startDate "YYYY-MM-DD"}},
+              {{format-date this.course.startDate}},
```

Comment:
```suggestion
              {{format-date this.course.startDate day="2-digit" week="2-digit" year="numeric"}},
```

see https://github.com/ilios/ilios/issues/5301

Reply:
I accept this. Corrected this to `day="2-digit" month="2-digit" year="numeric"` (and not `week`) and ✅